README.md

```diff
@@ -66,6 +66,18 @@ Per-VDOM:
  * `fortigate_link_bandwidth_rx_byte_per_second`
  * `fortigate_link_status_change_time_seconds`
 
+ Per-SDWAN and VDOM:
+ * `fortigate_virtualwan_healthcheck_status`
```

Comment:
Are there any other sub categories under virtual wan? As I unterstand it healthcheck is here only a reference to the API endpoint, right? 

If yes, I think `fortigate_virtual_wan_*` would be a better fit in the current metric naming style.    
Further can you add the suffixes of the `fortigate_link_*` metrics?  These names follow the Prometheus best practices.